src/CommunityCommons/javasource/myfirstmodule/actions/Test_Clone_Java.java

```diff
@@ -0,0 +1,59 @@
+// This file was generated by Mendix Studio Pro.
+//
+// WARNING: Only the following code will be retained when actions are regenerated:
+// - the import list
+// - the code between BEGIN USER CODE and END USER CODE
+// - the code between BEGIN EXTRA CODE and END EXTRA CODE
+// Other code you write will be lost the next time you deploy the project.
+// Special characters, e.g., é, ö, à, etc. are supported in comments.
+
+package myfirstmodule.actions;
+
+import com.mendix.systemwideinterfaces.core.IContext;
+import com.mendix.webui.CustomJavaAction;
+import java.util.UUID;
+import communitycommons.ORM;
+import system.proxies.FileDocument;
+
+public class Test_Clone_Java extends CustomJavaAction<java.lang.Boolean>
+{
+	public Test_Clone_Java(IContext context)
+	{
+		super(context);
+	}
+
+	@java.lang.Override
+	public java.lang.Boolean executeAction() throws Exception
+	{
+		// BEGIN USER CODE
+        FileDocument doc = new FileDocument(getContext());

```

Comment:
There's a mix of spaces and tabs for indentation, which makes it look sloppy. Please use one or the other.

Reply:
Fixed